draft-ietf-quic-tls.md

```diff
@@ -778,6 +778,21 @@ the connection that is hosted on stream 1.  This sequence number is not visible
 to QUIC.
 
 
+## Receiving Protected Packets
+
+Once an endpoint successfully receives a packet with a given packet number, it
+MUST discard all packets with higher packet numbers if they cannot be
+successfully unprotected with either the same key, or - if there is a key update
+- the next packet protection key (see {{key-update}}).  Similarly, a packet that
```

Comment:
This can only work if these packets were queued previously (because they were undecryptable). Otherwise, they might already have been processed.